extra-dev/packages/coq-test/coq-test.dev/opam

```diff
@@ -0,0 +1,22 @@
+opam-version: "2.0"
+name: "coq-test"
+version: "dev"
```

Comment:
This is redundant with the package file name. I would remove these to avoid future inconsistencies.

Reply:
Yes would be cool, thinking about it also, did not have time to make it yet.